### DISCUSSION TITLE
NFT到底是甚麼

### DISCUSSION THREAD

**Discussion by Tomlord1122** [1101-GS4719-C/Python-Community-GS4719-C#188] [科技新知]
NFT，又叫做非同質化代幣（Non-fungible token，簡稱「NFTs」），目前主要是應用在電子藝術品上面，可以證明買家確實擁有與特定數位藝術品、音樂或其他作品相連結的無形標記，有了這個應用，我們可以更加確保一個作品的交易歷史，出自於誰，誰曾經擁有過它，而現在有很多看好這項技術的投資人常常會使用虛擬貨幣，像是比特幣、以太幣來交易購買一些虛擬藝術品，我覺得雖然大家還沒有很了解買這些不存在於自然世界而存在網路世界裡的東西很莫名其妙，但其實誰都不能保證幾年後，科技的進步又會帶來甚麼重大的改變，但從虛擬貨幣的流行，到現在又有了NFT這種確保所有權的技術，更打好了metaverse的基礎，或許再過個一兩年，虛擬土地會是一個我們以後置產投資的方向，畢竟地球就這麼大，漸漸人們會把注意轉向虛擬世界吧

**Comment by Jie2222** [comment]
對實現metaverse來說，最重要的兩個東西就是虛擬貨幣跟NFT，網路上很多相關資訊和介紹，附上老高的影片當作NFT懶人包
[不可不知的NFT，元宇宙的根本，另一個世界的證明](https://www.youtube.com/watch?v=cDk1FPoCfqI)

**Comment by Jen-Siang** [comment]
虛擬世界的發展勢在必行，相信很快就會融入大家的生活中

**Comment by vvvioletta** [comment]
對於NFT，我有另一個想法提供給大家~
身邊許多朋友是為了追隨某個在幣圈有影響力的人而去買他發行的NFT，像是前陣子陳零九也開始買的FOMO dog，那個發行人是很有名的德州撲克選手，後來在幣圈也小有名氣(他有一個podcast叫優勢人生，有興趣的人可以去聽看看~) FOMO dog地板價在這兩個多月以指數狂飆，代表這個社群裡的人份量肯定不容小覷
以我來看，買NFT更像是買一張進入社群的門票，擁有那個NFT才能擠進你想認識的大老所屬的社群(當然也不只幣圈，也可以是藝術、音樂之類的社群)
我說的不一定正確，歡迎有其他想法的人一起交流~~~

**Reply by Tomlord1122** [reply]
沒錯，我覺得買NFT其實是背後要有一些利益才買的，不單單只因為錢，優勢人生的podcast我有聽過，德州撲克選手Raymond Wu，這裡也推薦一個podcast頻道，程人頻道，裡面也介紹很多幣圈的知識，還有工程師的求職分享，自我成長之類的，蠻推薦給大家的。

買NFT進入社群是一個現在在圈內蠻流行的事情，因為在這個不是大起就是大跌的環境裡，誰能快速掌握到最新的技術、項目，誰就能更能賺錢，最近還有NFT雞排跟NFT雞肉飯(這個我換算過了，很不划算)，雖然我覺得挺白癡的，但說實話還是很酷。

**Comment by yangming881005** [comment]
NFT真的有蠻多應用的地方，例如藝術品、入場券、遊戲token、抽獎盲盒等等，我覺得目前大家買NFT蠻大部份是在買個信仰，很多就單純只是個圖片而已XD，但也有蠻多公司在研究NFT的附加價值，像是前陣子Adidas發售的NFT，買家除了獲得NFT之外，也能在現實中獲得實體的東西，而遊戲項目方則是運用來把遊戲世界中的各種東西轉換成NFT，讓大家在玩遊戲的時候是真正擁有那個角色或道具，也能在遊玩遊戲的時候獲得他們發行的幣，來達到play to earn的目的。

**Comment by Isongzhe** [comment]
我個人覺得NFT如果要應用到遊戲界的話，比較適合拿來賣一些像是美術設定集，遊戲原始稿等等。跟遊戲實際內容不相關的

**Comment by chupochun** [comment]
最近真的好多人都在討論NFT 謝謝版主的介紹

**Comment by xkllkx** [comment]
謝謝樓主的介紹，最近NFT市場真的過度狂熱了。

**Comment by Ku-Hao-Chen** [comment]
謝謝分享，讓我對NFT有初步的認識。

**Comment by bloodtrail0511** [comment]
目前網路上有不少可供選擇的NFT交易平台，像是OpenSea、Nifty Gateway或SuperRare等等
